en/docs/monitoring-and-insights/view-logs.md

```diff
@@ -0,0 +1,106 @@
+# View Logs
+
+The unified log view in Choreo allows you to gain application and user insights while ensuring data privacy. 
+
+Choreo provides real-time insights through live logs and allows you to view historical logs for insights into the past. You also have the flexibility to define a required time window to view relevant log entries, where an intuitive scrolling capability facilitates easy viewing of appropriate logs within the selected time frame.
```

Comment:
Shall we add to say what type of logs we can view?

Reply:
+1
Rephrased the first sentence to indicate that Choreo provides runtime and audit logs.